xbmc/windowing/windows/WinKeyMap.h

```diff
@@ -15,164 +15,165 @@
 #include "settings/AdvancedSettings.h"
 #include "settings/SettingsComponent.h"
 
+#include <array>
+
 namespace KODI
 {
-  namespace WINDOWING
-  {
-    namespace WINDOWS
-    {
-      static XBMCKey VK_keymap[XBMCK_LAST];
+namespace WINDOWING
+{
+namespace WINDOWS
+{
+static std::array<XBMCKey, XBMCK_LAST> VK_keymap;
```

Comment:
As is more in namespace, maybe a new line above.

Reply:
added